mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -250,6 +246,7 @@ class VizServlet extends ScalatraServlet {
               val filteredData: RDD[(ReferenceRegion, CalculatedAlignmentRecord)] =
                 AlignmentRecordFilter.filterByRecordQuality(dataOption.get.toRDD(), readQuality)
               val binSize = VizUtils.getBinSize(region, VizReads.screenSize)
+              val unfilteredAlignmentData: Map[String, Array[ReadJson]] = AlignmentRecordLayout(dataOption.get.toRDD(), sampleIds)
```

Comment:
It's not used anywhere currently, but I was thinking when we were ready to send to the front end, we'd use it. Currently, filteredData is what we actually use, and that is passed into MergedAlignmentRecordLayout object, while I use just AlignmentRecordLayout object.